frontend/src/scenes/session-recordings/playlist/SessionRecordingsPlaylist.tsx

```diff
@@ -212,16 +220,26 @@ export function RecordingsLists({
                         </>
                     }
                     subheader={
-                        showFilters ? (
-                            <SessionRecordingsFilters
-                                filters={filters}
-                                setFilters={setFilters}
-                                showPropertyFilters={!personUUID}
-                                onReset={totalFiltersCount ? () => resetFilters() : undefined}
-                                usesListingV3={listingVersion === '3'}
-                            />
-                        ) : null
+                        <>
+                            {showFilters ? (
+                                <SessionRecordingsFilters
+                                    filters={filters}
+                                    setFilters={setFilters}
+                                    showPropertyFilters={!personUUID}
+                                    onReset={totalFiltersCount ? () => resetFilters() : undefined}
+                                    usesListingV3={listingVersion === '3'}
+                                />
+                            ) : null}
+                            <div className={'flex flex-row border-b items-center justify-center py-2 px-4 space-x-2'}>
+                                <span>Show</span>
+                                <DurationTypeSelect
+                                    value={durationTypeToShow}
+                                    onChange={(value) => setDurationTypeToShow(value)}
+                                />
+                            </div>
```

Comment:
Personally I really don't like this positioning. Looks super out of place. I don't have an immediate great alternative but I would lean towards having it either as part of the filters or as a submenu of the list.

Apart from being a lonely button it's not great that you have to scroll up to the top to change the value.
I also only really think that "active" or "total" are interesting to people. That could allow us to have a simple checkbox rather than a select, either allowing us to have one icon in the top bar or do it via a <More> dropdown which would feel much cleaner and inline with the rest of the UI.

Reply:
My inner Ben said "this should be in filters" too... I'll move it there.